frontend/src/lib/components/QuestionOptions.svelte

```diff
@@ -8,37 +11,105 @@
     }
 </script>
 
 <!-- We can add different types of question options 
     (like matching questions or fill in the blank) here in the future -->
+
 <form on:submit={check_answer}>
     {#each question.options as option, index}
-        {#if question.correctAnswer.length > 1}
-            <input
-                type="checkbox"
-                id={index.toString()}
-                bind:group={selected_answer}
-                value={index}
-            />
+        <!-- this div is displayed if the form has been submitted and the answer is correct -->
+        {#if correct.includes(index)}
+            <div class="m-3 w-1/2 rounded-lg bg-green-200 p-3">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={false} />
+            </div>
+
+            <!-- this div is displayed if the form has been submitted and the answer is incorrect -->
+        {:else if incorrect.includes(index)}
+            <div class="m-3 w-1/2 rounded-lg bg-red-200 p-3">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={false} />
+            </div>
+
+            <!-- this div is displayed if the form has not been submitted and the option is selected -->
+        {:else if selected_answer.includes(index)}
+            <div class="m-3 w-1/2 rounded-lg bg-slate-300 p-3">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={true} />
+            </div>
+
+            <!-- this div is displayed if the form has not been submitted and the option is not selected yet -->
         {:else}
-            <input type="radio" id={index.toString()} bind:group={selected_answer} value={index} />
+            <div class="m-3 w-1/2 rounded-lg p-3 hover:bg-slate-300">
+                <DisplayOption {option} {index} correct={question.correctAnswer} checked={false} />
+            </div>
         {/if}
-        <label for={index.toString()}>{option}</label>
-        <br />
     {/each}
-    <button type="submit">Submit</button>
+
+    <!-- this is the submit button -->
+    <button
+        class="m-3 inline-flex items-center gap-x-2 rounded-lg border border-gray-200 px-4 py-3 text-sm font-semibold text-gray-500 hover:border-blue-600 hover:text-blue-600 disabled:pointer-events-none disabled:opacity-50 dark:border-gray-700 dark:text-gray-400 dark:hover:border-blue-600 dark:hover:text-blue-500 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600"
+        type="submit">Submit</button
+    >
+    <br />
+
+    <!-- this is the back button -->
+    <button
+        class="m-3 inline-flex items-center gap-x-2 rounded-lg border border-gray-200 px-4 py-3 text-sm font-semibold text-gray-500 hover:border-blue-600 hover:text-blue-600 disabled:pointer-events-none disabled:opacity-50 dark:border-gray-700 dark:text-gray-400 dark:hover:border-blue-600 dark:hover:text-blue-500 dark:focus:outline-none dark:focus:ring-1 dark:focus:ring-gray-600"
```

Comment:
We have a dedicated pcss file to store all the reusable tailwindcss. You can make a new class called `navigation-button-style` like we do in `Button.pcss` in `styles` folder, and copy and paste this piece of css to there.